### PR TITLE
[minor] Optional Watson Studio bind to Health

### PR DESCRIPTION
**Pull request by alequint**
Following https://github.com/ibm-mas/ansible-devops/pull/957 - two main points about this PR:

- Update manage playbook offering possibility to install cp4d and studio to be bound with Health
- Modified he binding parameter (where we would pass a free string, expecting `system`) to a boolean flag. Reason for that is because `cp4d_service`role, today, when installing Studio, does not give us an option to create cfg in a different level inside MAS, only `system` is applicable (check [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_service/tasks/gencfg/gencfg-wsl.yml#L145)); same way, would not make any sense offer a choice at this point (if we want to bind, `True`, it will be a `system` binding)

### CODE REVIEW

File: ibm/mas_devops/roles/suite_app_config/README.md [hunk 1]
```diff
@@ -129,6 +129,13 @@ URL to access WML service (same as Cloud Pak for Data URL).
 
 Role Variables - Manage Workspace
 -------------------------------------------------------------------------------
+
+### mas_appws_bindings_health_wsl_flag
+Optional. Boolean value indicating if Watson Studio must be bound to Manage. It is expected a sytem level WatsonStudioCfg applied in the cluster.
```
Comment (andrercm) [inline ibm/mas_devops/roles/suite_app_config/README.md:134]: minor typo `sytem`

**Review by andrercm — CHANGES_REQUESTED** [review]
just a minor typo found, other than that i think it's looking ok.

File: ibm/mas_devops/roles/suite_app_config/README.md (hunk 1 above)
Comment (alequint) [reply ibm/mas_devops/roles/suite_app_config/README.md]: done

**Review by andrercm — APPROVED** [review]
LGTM

**Pull request merged**